### ISSUE TITLE
Scraping directory (exclude sub directories)

### ISSUE THREAD

**Issue by lamvuong2019** [ResetCollection/support#80]
As an example for Sega Dreamcast, I will have a directory for Windows CE games with in the directory. Redream does not play them, so I want to add them to flycast, and they are only supported on the x64 version.

There is no way to exclude the sub directory or add a sub system for a console.

**Comment by ReCoJav** [comment]
Yeah, right now sub directories are always scanned... I'll see about adding an option on the scrape scan option to ignore sub directories...

But you can if you want have two separate directories and add them separately as the same system...

roms_folder
- dreamcast_flycast
- dreamcast_redream

And then you can add Dreamcast system in RESET and choose dreamcast_flycast folder, then add another Dreamcast system in RESET and choose dreamcast_redream.

You can also have the one system with all dreamcast games, set it to the emulator that supports the most games, and go through each other game and set its emulator by bringing up the game's menu and selecting "Set emulator"

and that process can be seen in the tutorial video I made for RESET at following timestamp:
[12:40](https://www.youtube.com/watch?v=mIgttRQLNkk&t=760s) - Set a different emulator for a game

Hope this helps! let me know!
